packages/hash/external-services/kratos/kratos.prod.yml

```diff
@@ -73,12 +72,12 @@ selfservice:
             - hook: session
 
     # Email sending diabled in production for now
-    # verification:
-    #   use: code
-    #   lifespan: 48h
-    #   # Set though SELFSERVICE_FLOWS_VERIFICATION_UI_URL
-    #   ui_url: http://localhost:3000/verification
-    #   enabled: true
+    verification:
+      use: code
+      lifespan: 48h
+      # Set though SELFSERVICE_FLOWS_VERIFICATION_UI_URL
+      ui_url: http://localhost:3000/verification
+      enabled: true
```

Comment:
How can this be `http://localhost:3000` in the production kratos config? Wouldn't it have to be the public frontend URL (so `https://alpha.hash.ai`)?

Reply:
We're overriding all these through env vars at this diff: https://github.com/hashintel/hash/pull/1697/files#diff-a20ffb69d9d206b0d1749faf0e84f23ae7a85e38d016c8e4692946ffca6f36de
I understand it's very unclear, but I did not like the idea of configuration for production not being centralized in one place